README.md

```diff
@@ -2,43 +2,21 @@
 [![Build Status](https://travis-ci.org/haskell/ghcup.svg?branch=master)](https://travis-ci.org/haskell/ghcup)
 [![license](https://img.shields.io/github/license/haskell/ghcup.svg)](COPYING)
 
-# GHC up
+Although [`cabal-install`](https://hackage.haskell.org/package/cabal-install) manages Haskell library dependencies and can select a specific version of `ghc` (see [`with-compiler`](https://cabal.readthedocs.io/en/latest/nix-local-build.html#cfg-flag---with-compiler)), it will fail if the required version of `ghc` is not available. Unfortunately, many operating systems do not offer a way to install specific versions of `ghc`.
```

Comment:
I don't like the wording. It's not cabal-installs job at all to manage ghc versions or be aware of specifities (unix philosophy). That's why I chose the wording down below.

Reply:
`cabal-install` **selects** the `ghc` version used by a project, it doesn't manage it. I can remove the word "manages" as it is overloaded.